cvat/settings/development.py

```diff
@@ -22,8 +22,8 @@
 UI_HOST = os.environ.get('UI_HOST', 'localhost')
 UI_PORT = os.environ.get('UI_PORT', 3000)
 CORS_ALLOW_CREDENTIALS = True
-CSRF_TRUSTED_ORIGINS = [UI_HOST]
 UI_URL = '{}://{}'.format(UI_SCHEME, UI_HOST)
+CSRF_TRUSTED_ORIGINS = [UI_URL]
```

Comment:
I think this needs to happen after `UI_URL` is modified to include the port (the port is part of the origin).